README.md

```diff
@@ -324,6 +324,13 @@ Emitted when a LevelDB query is made using the `leveldown` module.
     * `value` (Object) The value being added to the LevelDB database using the `put` method (Undefined for other methods) 
     * `opCount` (Number) The number of operations carried out by a `batch` method (Undefined for other methods) 
     * `duration` (Number) the time taken for the LevelDB query to be responded to in ms.
+    
+### Event: 'loopback-datasource-juggler'
+Emitted when a function is called on the `loopback-datasource-juggler` module
```

Comment:
@sjanuary Please raise the pull request updating README yourself if you don't mind.

> For the probe improvements you would also be welcome to raise a pull request, or if not then could you open a new issue to track the requirement? 

Makes sense. Looks like Sam has already a backlog item for this work, let's leave it at that for now.